### DISCUSSION TITLE
is local send available for 32 bit devices

### DISCUSSION THREAD

**Discussion by fayizhere** [localsend/localsend#1119] [Discussion (English, International)] [answered]
does the local send appliaction available for 32 bit

**Comment by Tienisto** [comment, marked as answer]
No, 32 bit is currently not supported.

**Reply by VityaSchel** [reply]
how to get files of an old 2005 laptop then
there is literally no way omfg

**Reply by VityaSchel** [reply]
> how to get files of an old 2005 laptop then
> there is literally no way omfg

I ended up using ftp

**Comment by Max9199** [comment]
Yes, a 32-bit version would be very useful for older laptops etc.

**Comment by sudo-joe** [comment]
Would appreciate a 32-bit version too :)

**Comment by KaijuKoder2** [comment]
Would appreciate a 32-bit version for Windows as well. Because localsend is used for moving files around, that would often involve moving from older 32-bit Windows PCs (old versions of Windows) to new 64-bit machines (new versions of Windows). So a 32-bit version would be very useful. localsend works so well moving files from anything to anything, it's a shame to leave such a large chunk of PCs which especially need transfer out.
